### ISSUE TITLE
Carousel moves in reverse direction on iOs safari browser

### ISSUE THREAD

**Issue by shahsawooditr** [twbs/bootstrap#39721]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

Actually I have been facing issue on the ios Safari browser that the carousel slides in reverse automatically while on other os and browsers it moves forward (as expected), either fix the issue or provide an option for sliding direction control.

### Reduced test cases

It is not applicable.

### What operating system(s) are you seeing the problem on?

iOS

### What browser(s) are you seeing the problem on?

Safari

### What version of Bootstrap are you using?

v5.3.2

**Comment by julien-deramond** [comment]
Thanks for reporting this issue @shahsawooditr 
Could you please precise what iOS and Safari versions so that we can try to reproduce it?
Can't reproduce with BrowserStack simulator right now with the latest versions

**Comment by Twinstream** [comment]
Describe the issue
iOS Safari 12 browser carousel slides in reverse after first slide while on other os browsers carousel moves forward.

Reduced test cases
Using Bootstrap Documentation onlne examples for testing. 5.1, 5.2, 5.3.2

What operating system(s) are you seeing the problem on?
iOS mobile

What browser(s) are you seeing the problem on?
Safari 12 Standard Settings

What version of Bootstrap are you using?
v5.3.2

The issue of the carousel going backwards after clicking on mobile the first slide appeared first in 5.2. 

5.3.2 the issue persists.

5.1 the carousel functions normally on Safari 12 Standard iOS

If "Pointer Events" in "Experimental Webkit Features" is enabled in Safari 12 iOs settings, then the carousel functions correctly advancing the slide from 1 to 2 to 3 to etc.

Since browser support is listed as 5.3.2:

# https://github.com/browserslist/browserslist#readme

>= 0.5%
last 2 major versions
not dead
Chrome >= 60
Firefox >= 60
Firefox ESR
iOS >= 12
Safari >= 12
not Explorer <= 11

then it is factually incorrect and either the list needs to update saying Safari >= 13 or the js revisited ?

**Comment by julien-deramond** [comment]
Thanks @Twinstream for providing the problematic browser/platform combination. I can confirm the issue with BrowserStack with iOS and Safari 12. Also, confirm that the issue starts to appear with the v5.2 (works with v5.1.3).
Clicking "Next" on the first slide goes to the second slide, clicking "Next" on this second slide goes back to the first slide, and then the carousel works backward from there.

**Comment by shahsawooditr** [comment]
@julien-deramond is the issue fixed?

**Comment by julien-deramond** [comment]
> @julien-deramond is the issue fixed?

No, the issue is still open. It will be closed once it's fully addressed.

**Comment by julien-deramond** [comment]
> Is there anything from my side that needs to be addressed or from the core team?

Based on the history of the discussion, I don't think so. We (or someone from the community) just need to find time to check this more in depth and provide a patch :)

**Comment by mdo** [comment]
v6-dev rebuilds the carousel on native CSS scroll-snap, so the v5 transform/`deltaX` touch code behind the reversed iOS swipe is gone (and Safari 12 is below v6's browser baseline). Closing as resolved in v6-dev — please open a new issue if you see reversed swiping on a current iOS Safari there.

**Comment by Kevinjohn** [comment]
I'm hijacking this a little (sorry) because I've a [PR](https://github.com/twbs/bootstrap/pull/42714) in for iOS 12, with replicable tests, that I think makes this a candidate for [5.4](https://github.com/orgs/twbs/projects/12/views/1)

Appreciate that iOS12 won't be supported by v6, but to me, that means even more reason to get this fix in.

I might be getting on my soapbox here, as someone who has hand-me-down devices to my kids (and parents), but the last iOS 12 devices are only 6 years old - seems a shame to not fix this, when they're technically on the browser support list.

Would you like me to open another Issue for v5, keeping this one for v6 (for ease of project management)?
